### DISCUSSION TITLE
osu!lazer: left click key not working at all

### DISCUSSION THREAD

**Discussion by DluX-Summer** [ppy/osu#17618] [Q&A] [answered]
I normally use Y and X buttons on the keyboard to tap, and it works perfectly fine on classic osu. However, on lazer, i could only tap with Y, X wouldnt work at all. Key bindings are set up correctly.

I thought this maybe was an issue with keyboard layouts (im using german layout) so i switched to using S and D keys. Same problem here, the D key works perfectly fine, the S would refuse completely. Fiddling around a little bit, i discovered that whatever i set as "Left click" in key bindings will refuse to work completely. If i flip my key binding settings, S works, D refuses. This happens on any map and with any set of keys on the keyboard.

Could there be an issue or am i doing something wrong?

Sincerely,
d0msen

**Comment by integerrr** [comment]
please post a screenshot of your keybinds and perhaps a video of the issue with the key counter on in gameplay

**Comment by DluX-Summer** [comment, marked as answer]
So turns out this is indeed my fault. Putting on the key counter i noticed my left key was pressed the entire time. This is due to me dragging on the tablet, so it activates the mouse button wich i thought i had disabled. Removing key binding for m1 and m2 or disabling mouse button while playing makes it work just fine.

Thanks for reaching out though!
